Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -118,43 +106,43 @@ public static class SystemInformation
         public static PackageVersion FirstVersionInstalled { get; }
 
         /// <summary>
-        /// Gets the DateTime (in UTC) when the app was launched for the first time
+        /// Gets the DateTime (in UTC) when the app was launched for the first time.
         /// </summary>
         public static DateTime FirstUseTime { get; }
 
         /// <summary>
-        /// Gets the DateTime (in UTC) when the app was previously launched, not including this instance.
-        /// Will be DateTime.MinValue if `TrackAppUse` has not been called.
+        /// Gets the DateTime (in UTC) when the app was last launched, not including this instance.
+        /// Will be <see cref="DateTime.MinValue"/> if <see cref="TrackAppUse"/> has not been called yet.
         /// </summary>
         public static DateTime LastLaunchTime { get; private set; }
 
         /// <summary>
-        /// Gets the number of times the app has been launched since the last reset.
```

Comment:
>Gets the number of times the app has been launched **since the last reset**

better with or without?

Reply:
It's clear that a reset actually resets the count to zero, so without should be fine.